client/components/mma/cancel/tierThree/TierThreeCancellationReasons.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+	inOrderToImproveSubs,
+	standardAlternateFeedbackIntro,
+} from '../cancellationConstants';
+import type { CancellationReason } from '../cancellationReason';
+import { BreakFromNewsWithAlternative } from '../GenericSaveBodyResponses';
+
+export const tierThreeCancellationReasons: CancellationReason[] = [
+	{
+		reasonId: 'mma_editorial',
+		linkLabel: 'I am unhappy with Guardian journalism',
+		saveBody: [
+			'In order to improve our journalism, we’d love to know more about why you are thinking of cancelling.',
+		],
+		alternateFeedbackIntro: standardAlternateFeedbackIntro,
```

Comment:
is this making the "custom feedback form prefix message" of "Please share any further thoughts you have about cancelling — you can help us improve. Thank you."

Reply:
yes it is
https://github.com/guardian/manage-frontend/blob/16068c6661741d982c140afe39e54711bffe2e92/client/components/mma/cancel/cancellationConstants.ts#L3-L4

Reply:
yep